### PR TITLE
remove helm2 support

### PR DESCRIPTION
**Pull request by manojn97**
# Pull Request Review

## Checklist

- [x] Issue: https://github.ibm.com/TNC/tnc-o-tracking/issues/3394
- [x] List of related PRs. : None
- [x] Project builds without any errors.
- [x] Unit Tests updated (or created where needed) and all pass.
- [x] You have ran manual functional “smoke” test (does product as a whole still work?).
- [x] User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
- [x] There is a PR for update of internal documentation for the affected Project(s) - README etc.
- [x]  **Description of the changes**:   https://github.ibm.com/TNC/tnc-o-tracking/issues/3394

### CODE REVIEW

File: docs/reference/associated-topology.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ id: Helm:docs:example
 type: Helm
 ```
 
-The driver uses the name of the Helm release and the namespace it was installed in as a unique id and name (even though we are using Helm2, this prepares us for Helm3 where releases can have the same name but in different namespaces). 
+The driver uses the name of the Helm release and the namespace it was installed in as a unique id and name (this prepares us for Helm3 where releases can have the same name but in different namespaces).
```
Comment (hhellyer) [inline docs/reference/associated-topology.md:100]: The whole of the text about preparing for helm 3 can be removed here. The preparation is done.

**Review by owen-lynch-ibm — DISMISSED** [review]
Looks a lot better than before with updates as recommended by @hhellyer applied. I'm happy to approve, but make sure to address outstanding comments from Howard.

**Review by hhellyer — DISMISSED** [review]
LGTM - I checked the test results in the associated issues and they look good. I can't see any references now to helm 2 or the specific version of helm 2 we used. There is one comment that is worth updating in docs/reference/associated-topology.md but once that is tidied this can be merged.

File: docs/reference/associated-topology.md (hunk 1 above)
Comment (manojn97) [reply docs/reference/associated-topology.md]: Hi @hhellyer , I have removed the text as per your comments

**Pull request merged**